docs/adminguide/frozen_users.rst

```diff
@@ -0,0 +1,114 @@
+
+A successful request returns a JSON dictionary similar to the one below:
+
+  .. code-block:: json
+
+    {
+      "users": [
+        {
+          "user_name": "Alice",
+          "frozen": true,
+          "user_email": "alice@example.com",
+          "user_id": "940a380aedd44127863d952a66cfce1e"
+        },
+        {
+          "user_name": "Bob",
+          "frozen": false,
+          "user_email": "bob@example.com",
+          "user_id": "7d04b0df51a74158b35b6386eecdf4e0"
+        }
+      ]
+    }
+
+HTTP ``users/frozen`` route
+---------------------------
+
+This route is made available as ``/administration/organizations/<raw_organization_id>/users/frozen`` and requires an administration token.
+
+It only supports the ``POST`` method which modifies the ``frozen`` status for a given user.
+
+Here's an example of generating the request data using ``jq``:
+
+  .. code-block:: bash
+
+    $ DATA=$(jq -n --arg user_id "$user_id" --argjson frozen true '$ARGS.named')
+    $ echo $DATA
+    {
+    "user_id": "940a380aedd44127863d952a66cfce1e",
+    "frozen": true
+    }
+
+The request can also use the ``user_email`` field instead of ``user_id`` to identify the Parsec user:
+
+  .. code-block:: bash
+
+    $ DATA=$(jq -n --arg user_email "$user_email" --argjson frozen true '$ARGS.named')
+    $ echo $DATA
+    {
+    "user_email": "alice@example.com",
+    "frozen": true
+    }
+
+Here's an example of running the request using ``curl`` and ``jq``:
+
+  .. code-block:: bash
+
+    $ curl https://$server/administration/organizations/$organization/users/frozen \
+        -H "Authorization: Bearer $administration_token" \
+        --request POST --data $DATA | jq
+
+A successful request returns a JSON dictionary similar to the one below:
+
+  .. code-block:: json
+
+    {
+      "frozen": true,
+      "user_email": "alice@example.com",
+      "user_id": "940a380aedd44127863d952a66cfce1e",
+      "user_name": "Alice"
+    }
+
+User identification: email address vs Parsec ID
+-----------------------------------------------
+
+There is a subtle difference between the two ways to identify a user. At any given time, an email address can be used to uniquely identify a non-revoked user from a given organization. In contrast, a Parsec user ID identifies uniquely any user from all organizations in the Parsec server, including revoked users. This means that over time, an email address can identify different Parsec users with different Parsec IDs, even from the same organization.
+
+The frozen status configured by the ``POST`` method is specifically associated with the Parsec user ID, regardless of the identification method used in the request body. This has the following consequence: if a user is revoked and then a new user is created with the same email address, the frozen status will **not** be applied to the new user.
+
+Error handling
+--------------
+
+The following standard errors are handled the same way as for the other administration routes:
```

Comment:
```suggestion
The following standard errors are handled the same way as for other administration routes:
```

Reply:
Actually it's weird to talk about "other administration routes" since (i think) this is the very first time we publicly document an administration route.

Reply:
[70d6ef2](https://github.com/Scille/parsec-cloud/pull/5842/commits/70d6ef29491522ed83c28a1dcd9f084b78896ef2)